cmd/crc/cmd/start.go

```diff
@@ -250,3 +256,25 @@ func commandLinePrefix(shell string) string {
 	}
 	return "$"
 }
+
+func checkDaemonStarted() error {
+	if config.Get(cmdConfig.NetworkMode).AsString() == string(network.DefaultMode) {
+		return nil
+	}
+	daemonClient := daemonclient.New()
+	version, err := daemonClient.APIClient.Version()
+	if err != nil {
+		return fmt.Errorf(daemonStartedErrorMessage(), err)
+	}
+	if version.CrcVersion != crcversion.GetCRCVersion() {
+		return fmt.Errorf("The executable version (%s) doesn't match the daemon version (%s)", crcversion.GetCRCVersion(), version.CrcVersion)
+	}
+	return nil
+}
+
+func daemonStartedErrorMessage() string {
+	if crcversion.IsMacosInstallPathSet() {
+		return "Is '/Application/CodeReady Containers.app' running? Cannot reach daemon API: %v"
```

Comment:
It's `/Applications` (missing 's'). If this message is intended for users, should it be using "macOS tray" instead? But I don't think it's going to be possible to trigger this from the tray, as if the daemon is not running, we'll never reach the go code?

Reply:
Thanks fixed. 
This message is for people trying to run `crc` from the CLI without the tray launched. "macOS tray" doesn't really say what do to to get this working.